go/grpcweb/grpc_web_response.go

```diff
@@ -18,10 +21,29 @@ type grpcWebResponse struct {
 	wroteBody    bool
 	headers      http.Header
 	wrapped      http.ResponseWriter
+
+	// Data in the body of the response must be written to writer.
+	writer io.Writer
```

Comment:
With the addition of this, what are we still using the `wrapped` field for? Can we shrink its interface definition from `http.ResponseWriter`?

Reply:
Hm, in my mind the Base64ResponseWriter would seem nice here, as opposed to two writers I think. Lets try that if you don't mind?

Reply:
I gave this a shot. This ended up changing the output slightly, but in a way that is supported at least by the official grpc-web client (my personal "interop test" doesn't use the client in this repo).

Reply:
Also I should note: thanks for pushing on this, I think the end result actually is cleaner.